### DISCUSSION TITLE
Outline View Heading Icons

### DISCUSSION THREAD

**Discussion by Arbiter313** [notaapp/talk#292] [Feature Requests] [answered]
I am really enjoying the new update and especially the new outline view! I like the subtle indenting of the headings levels in the outline too. However, the '#' icon next to each heading cap be confusing, especially if there are a lot of headings. It would be great to see numbered icons next to each heading in the outline instead, indicating what level of heading it is, similar to the small H1, H2, etc. used in the editor?

**Comment by erusev** [comment, marked as answer]
That makes sense, I'll see if we can fix it for the September release. Thanks, and I'm glad that you like the update 😊

**Reply by Arbiter313** [reply]
In their new release Obsidian has a really nice design for displaying headings in the outline view as seen in the attached image. It makes the hierarchical relationships very clear and has the added advantage of fitting the style of how some of the hierarchical elements are displayed in the Nota editor anyway. They levels collapse too which I found useful!

The only improvement to it I would make is the suggestion I already mentioned in the above post, namely replacing the downward-facing arrowheads in the image with the appropriate numbered heading icon. 

<img width="752" alt="headings-outline" src="https://user-images.githubusercontent.com/45447185/195722304-92ce89d3-88d1-4f53-ace4-1b8f2207b66a.png">

**Reply by erusev** [reply]
I agree, thanks for taking the time to share this 🙏

**Reply by petrbouchal** [reply]
Yes, a nested outline would be great!

**Comment by erusev** [comment]
We just released an update (`0.40`) where we tried to address this. To get it, select Nota (from the app menu) → Check for Updates...
